diff --git a/conf/server/server.conf b/conf/server/server.conf
index a21d53b9..b70fd76a 100644
--- a/conf/server/server.conf
+++ b/conf/server/server.conf
@@ -3,7 +3,7 @@ server {
     bind_port = "8081"
     socket_path = "/tmp/spire-server/private/api.sock"
     trust_domain = "example.org"
-    data_dir = "./.data"
+    data_dir = "/tmp/data"
     log_level = "DEBUG"
 }

@@ -11,7 +11,7 @@ plugins {
     DataStore "sql" {
         plugin_data {
             database_type = "sqlite3"
-            connection_string = "./.data/datastore.sqlite3"
+            connection_string = "/tmp/data/datastore.sqlite3"
         }
     }

@@ -26,8 +26,8 @@ plugins {

     UpstreamAuthority "disk" {
         plugin_data {
-            key_file_path = "./conf/server/dummy_upstream_ca.key"
-            cert_file_path = "./conf/server/dummy_upstream_ca.crt"
+            key_file_path = "/etc/spire/server/dummy_upstream_ca.key"
+            cert_file_path = "/etc/spire/server/dummy_upstream_ca.crt"
         }
     }
 }
